.kokoro/functions/functions-billing.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2019, Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+      
+export GCLOUD_PROJECT=cdpe-functions-billing-test
+export GCP_PROJECT=$GCLOUD_PROJECT
+export GOOGLE_CLOUD_PROJECT=$GCLOUD_PROJECT
```

Comment:
Is this env variable being used?

Reply:
I believe it's used elsewhere - I was mainly adding it to save us the trouble if future samples start relying on it.